test/test_utils/sagemaker.py

```diff
@@ -183,8 +183,8 @@ def install_sm_local_dependencies(framework, job_type, image, ec2_conn):
     # Install custom packages which need to be latest version"
     is_py3 = " python3 -m"
     # To avoid the dpkg lock with apt-daily service if exists
-    sleep(300)
-    # using virtualenv to avoid package conflicts with the current packages            
+    ec2_conn.run(f"sleep 300")
```

Comment:
nit: why is this an f string?

Reply:
I wanted to make the sleep time a variable controlled from the python script, but wanted to try experimenting with 300 seconds hardcoded first. Will change it as soon as this attempt succeeds (which it looks like it will).